bbc/simorgh
# Add amp prop in Articles container & component

Resolves #583

Sets `amp` prop to true when the path matches the AMP route. `/news/articles/amp/:id`

Currently, this will just add the amp attribute to the `<html>` element. 

The work for using `<amp-img>` instead of `<img>` and other required AMP-compatibility work will be carried out in separate PRs. 

*Testing instructions*
```
git checkout amp-prop
npm install
npm run dev
```
When you visit an AMP route for an article page, (http://localhost:7080/news/articles/amp/c0000000027o) on the `<html>` element you should see the attribute `amp="true"`.
When you visit a standard article route, (http://localhost:7080/news/articles/c0000000027o) on the `<html>` element you should *not* see the attribute `amp`.
